keras2onnx/parser.py

```diff
@@ -735,7 +743,15 @@ def parse_graph_modeless(topo, graph, target_opset, input_names, output_names, k
 
     for ts_i_ in input_tensors:
         var_type = _adjust_input_batch_size(infer_variable_type(ts_i_, target_opset))
-        str_value = ts_i_.name
+        if ts_i_.name.endswith(':0'):
```

Comment:
Any special things in the model causes ':0'?

Reply:
This is tensor name so it has ":0". This code path is `parse_graph_modeless`, in the traditional path, we also have logic to handle this, see [here](https://github.com/onnx/keras-onnx/blob/89b0c459ea2dd1aa086d16983d70547992ef8095/keras2onnx/parser.py#L775-L785)